com.discordapp.Discord.appdata.xml

```diff
@@ -44,14 +46,30 @@
       <caption>Available on Linux, macOS, Windows, iOS, Android, and your web browser</caption>
     </screenshot>
   </screenshots>
-  <kudos>
-    <kudo>HiDpiIcon</kudo>
-    <kudo>Notifications</kudo>
-  </kudos>
+  <branding>
+    <color type="primary" scheme_preference="light">#a0aaf3</color>
+    <color type="primary" scheme_preference="dark">#3831a8</color>
+  </branding>
   <categories>
     <category>InstantMessaging</category>
     <category>Network</category>
   </categories>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-info">intense</content_attribute>
+    <content_attribute id="social-audio">intense</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+    <content_attribute id="money-purchasing">intense</content_attribute>
+  </content_rating>
+  <requires>
+    <display_length compare="ge">940</display_length>
+    <internet>always</internet>
+  </requires>
+  <supports>
+    <control>keyboard</control>
+    <control>pointing</control>
+  </supports>
+  <update_contact>tingping_at_fedoraproject.org</update_contact>
```

Comment:
Since this package has been "official" for a while, I think it makes sense to replace this email address with their [official one](https://discord.com/company-information) instead: `support@discordapp.com`.

Reply:
Sure thing :)